blockchain/fundamentals/ethereum-on-the-web/README.md

```diff
@@ -7,5 +7,6 @@ description: todo
 section: 2
 
 insights:
-  - web3js
+  - metamask
+  - myetherwallet
```

Comment:
any chance we could have another insight here? asking because an workout need min 3 insights to be chosen in the bot

Reply:
sure thing, ill add the 3rd one